### PR TITLE
feat: support flyte decks

### PR DESCRIPTION
**Pull request by james-union**
https://github.com/flyteorg/flyteconsole/issues/469

https://user-images.githubusercontent.com/105876962/172195585-f42ec187-a68f-4d9c-a78b-e0dc8ba6b659.mov

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
- Added Flyte Deck button
- Show Flyte Deck button if only deckUri is valid
- Enable Flyte Deck button if only the execution is succeed
- Get Signed url of deck html from the native url (deckUri)
- Show html in iframe
- Fix python template style. (on the backend side with Kevin) 

https://github.com/flyteorg/flyte/issues/469

### CODE REVIEW

File: packages/zapp/console/src/components/Executions/ExecutionDetails/ExecutionDetailsActions.tsx
Comment (anrusina) [inline packages/zapp/console/src/components/Executions/ExecutionDetails/ExecutionDetailsActions.tsx:132]: nit: Ideally we want to use Typography with h2 and h4 here.
Hobnxgun created a design system to be used for FlyteConsole and we slowly moving to use it, but these heading are not yet in place:
https://www.figma.com/file/D96LNTCFgt5TDfEhePdTMU/Flyte-Console?node-id=6177%3A1458255
If it's need to be merged ASAP -> fee free to ignore this need, but still update headings h3 and tab names to a proper heading (h2/h4) usage to simplify migration in future
Screen Shot 2022-06-06 at 11 06 34 AM
Screen Shot 2022-06-06 at 11 06 43 AM
Screen Shot 2022-06-06 at 11 07 07 AM

File: packages/zapp/console/src/models/Common/constants.ts
Comment (anrusina) [inline packages/zapp/console/src/models/Common/constants.ts:17]: just curious, what is urn states for? as looks like it used in enpoint url too

**Review by anrusina — COMMENTED** [review]
Looks good. Haven't smoke tested, just briefly looked through the code. Will leave smoke test to Carina :)

File: packages/zapp/console/src/models/Common/constants.ts
Comment (james-union) [reply packages/zapp/console/src/models/Common/constants.ts:17]: Good question. I also had that question for Kevin 
Answer from Kevin
```
The location (s3://..) where we store deck.html or output.pb
```

**Comment by ursucarina** [comment]
lgtm :)

**Review by ursucarina — DISMISSED** [review]

**Comment by anrusina** [comment]
?? Taking in mind that you was needed to setup a different ADMIN_API_URL to access this data, do we need the different value in production too, or should it work out of the box?
If needed - I can deploy it to DEMO so we can later check it. 
cc: @jsonporter

**Comment by james-union** [comment]
> ?? Taking in mind that you was needed to setup a different ADMIN_API_URL to access this data, do we need the different value in production too, or should it work out of the box? If needed - I can deploy it to DEMO so we can later check it. cc: @jsonporter

We will wait until backend team merged into demo so that we won't do any custom config to test this out.

**Comment by james-union** [comment]
> @james-union for local dev, can we add ADMIN_API_USE_SSL in clientConfig.devServer.server.type?

Done

**Comment by james-union** [comment]
@jsonporter @anrusina @olga-union @ursucarina 
The backend is deployed to the demo and we can test without any env change.

Here is a test execution which contains deck & valid signed url
https://localhost.demo.nuclyde.io:3000/projects/flytetester/domains/development/executions/a95zrszx9w5mk4vf72sf?duration=all

**Review by jsonporter — APPROVED** [review]
lgtm

**Pull request merged**
